### ISSUE TITLE
Can I use links from

### ISSUE THREAD

**Issue by RHGF44** [UnamSanctam/UnamDownloader#71]
Hello.
1. Can I use links from https://bitly.com or similar services to download files?
2. Will the file be loaded via a redirect, for example, using iplogger?

**Comment by UnamSanctam** [comment]
Yes you should be able to download using redirects (such as bitly or an IPLogger).
